src/components/app-configure/AppKeyspace.vue

```diff
@@ -111,9 +111,6 @@ function handleCancel() {
         <div v-if="chosenWalletUIMode !== CUSTOM_UI" class="relative">
           <div
             class="keyspace-card"
-            :class="{
-              'disabled-card gradient-border-card': app.status !== 2,
-            }"
             @click.stop="
               app.status === 2 ? (selectedKeyspace = 'global') : void 0
```

Comment:
Need to update this logic as well

Reply:
click.stop="selectedKeyspace='global'"

Reply:
[done](https://github.com/arcana-network/developer-dashboard/pull/431/commits/2cfcff59411aab2d20a21eed12b045baf10e250e)